dnf-behave-tests/features/repoquery/globs.feature

```diff
@@ -0,0 +1,180 @@
+Feature: Glob tests for expanding all the various glob patterns.
+
+Background:
+ Given I use the repository "repoquery-globs"
+
+
+# <name> globs
+# TODO it's weird this doesn't actually work (lists nothing)
+@xfail
+Scenario: repoquery *
+ When I execute dnf with args "repoquery *"
```

Comment:
replace * with '*' to avoid shell expansion

Reply:
Duh.